src/Auth/helpers.js

```diff
@@ -44,7 +42,18 @@ async function blockValue(field){
     }
   }
 
+function getOwnerid(profileObject){
+  if (typeof profileObject !== "undefined"){
+      if (propertyExists(profileObject, "profile")){
```

Comment:
this function will always throw false because `"profile"` is not a property of `profileObject`.  The `profileObject` is essentially a Profile object so it's properties are `gcID, email, name,` ect.